colymore/woocommerce-android
# Add new tab for Analytics section

# Add new tab for Analytics section

### Issue
This pull request closes Issue [5236](https://github.com/woocommerce/woocommerce-android/issues/5236)

### Description
This pull request is going to add an item in the button navigation bar which will be the entry point for the new analytics hub. This is going to be set behind a feature flag to enable it only when we are ready.

### Testing instructions
For testing the feature we can just open the application and check the new icon for analytics appears only for debug applications and it is not appearing for non-debug applications so the feature flag would be closed.

### Result

![Screenshot 2021-11-19 at 18 34 59](https://user-images.githubusercontent.com/572601/142666634-d569480b-4e0f-40d7-8955-98133e399098.png)

![Screenshot 2021-11-19 at 18 34 44](https://user-images.githubusercontent.com/572601/142666627-18ce022d-a23d-40d8-84eb-7d61f0549b2f.png)
